step-automation-packages/step-automation-packages-manager/src/main/java/step/automation/packages/execution/IsolatedAutomationPackageRepository.java

```diff
@@ -177,6 +181,55 @@ public void cleanUpOutdatedResources() {
         log.info("Cleanup outdated automation packages finished. {} of {} packages have been removed", removed, foundResources.size());
     }
 
+    public void cleanUpMavenCache() {
+        log.info("Cleanup outdated artifacts in maven cache...");
+        String ttlString = ttlValueSupplier.get();
+
+        long ttlDurationMs = Long.parseLong(ttlString);
+        Long minExecutionTime = System.currentTimeMillis() - ttlDurationMs;
+
+        int removed = 0;
+        int artifactsCandidate = 0;
+        try {
+            Path mavenCachePath = Path.of("maven");
```

Comment:
If I'm not wrong this can be changed. If yes, we should read it from the config

Reply:
done